CHANGELOG.md

```diff
@@ -20,10 +20,14 @@ Keep the changelog pleasant to read in the text editor:
 
 version 2.0.0
```

Comment:
Should there be a `1.1.0` section in the changelog?

Reply:
Yes - I'll add it

Reply:
The question is what to put there? Do I move all the changelog items from 2.0 to 1.1? Copy them? Just say "the following items were backported from 2.0"?

Reply:
Since 2.0 doesn't yet technically exist - what about moving the backported items to just 1.1 and then 2.0 becomes the net-new beyond 1.1?

Reply:
Done.